capa/capabilities/static.py

```diff
@@ -118,6 +119,9 @@ def find_code_capabilities(
         features, bmatches, imatches = find_basic_block_capabilities(ruleset, extractor, fh, bb)
         for feature, vas in features.items():
             function_features[feature].update(vas)
+            if isinstance(feature, API):
+                # delcare a global variable (a set) and append to it here?
+                pass
```

Comment:
The goal here is to count how many API calls are made. I am thinking of declaring a global variable here (i.e. `Set[API]`) and appending to it?

Reply:
once we have all the features collected and merged at the file level, we could enumerate them once to collect the API features and count them. We shouldn't need a distinct variable for this.

Reply:
around like 231. 

Though it's possible we don't construct the massive set like we do at lower scopes, I can't quite remember.

Reply:
> once we have all the features collected and merged at the file level, we could enumerate them once to collect the API features and count them. We shouldn't need a distinct variable for this. 

You're right, we don't. We collect all the features from the smaller blocks, but for functions we only return the `len(function_features)`.